hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/tasks/TestOmTableInsightTask.java

```diff
@@ -155,6 +193,56 @@ public void testProcess() {
     assertEquals(5L, getCountForTable(DELETED_TABLE));
   }
 
+  @Test
+  public void testProcessForSize() {
```

Comment:
Write and update test cases in "org.apache.hadoop.ozone.recon.api.TestOmDBInsightEndPoint"  class